app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -53,15 +53,17 @@ environment
 
 ## Your classroom placements
 
-Typically, you will get 120 days of classroom experience in a minimum of two schools. Through your placements, you’ll have the opportunity to teach, plan, observe and research.
+You should get 120 days of classroom experience as part of a postgraduate or 3 year undergraduate teacher training course. On a 4 year undergraduate course, you should get 160 days of classroom experience. 
```

Comment:
I'm not sure it is right to talk about undergraduate training here - the rest of this page is about postgrad as far as I understand it (although we are probably not as explicit about that as we could be).

cc @EllieNodder for thoughts?

Reply:
That's a good point - we talk about teacher training taking 9 months full-time and 18 to 24 months part-time which is obviously in reference to postgrad ITT, and talk about the assessments which are specific to postgrad training.

I think in this case, the page is just about postgrad training which we could stand to be more clear about so probably don't want to mention undergrad here (and we should also probably revisit this same issue around the website, because I think we're often so focused on postgrad that we assume the user knows that's what we're talking about, which might not always be the case - but that's an issue for another time ...)